src/kubernetes.cc

```diff
@@ -1034,8 +1061,15 @@ void KubernetesReader::WatchNode(MetadataUpdater::UpdateCallback callback)
   LOG(INFO) << "Watch thread (node) exiting";
 }
 
-void KubernetesUpdater::start() {
-  PollingMetadataUpdater::start();
+bool KubernetesUpdater::ValidateConfiguration() const {
+  if (!PollingMetadataUpdater::ValidateConfiguration()) {
```

Comment:
Should this not also be in `DockerUpdater` and `InstanceUpdater`?

Reply:
Forgot about `DockerUpdater`, `InstanceUpdater` does not override this method.